markdown/org/docs/designs/opal/fabric/en.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Opal Overalls: Fabric Options"
+---
+
+This pattern is designed to work with denim, but should work with most woven fabrics.
+
+Fasteners are generally used to attach the straps to the front bib. I make my overalls using tack buttons on the bib, and a buckle and slider on each strap.
```

Comment:
This information might be better in need than fabric

Reply:
Moved.